.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+jobs:
+  gnu_mpich:
+    docker:
+      - image: dashproject/ci:mpich
+    environment:
+      DASH_MAKE_PROCS: 2
+      DASH_MAX_UNITS: 3
+      DASH_BUILDEX: OFF
+      CC: gcc
+      CXX: g++
+    steps:
+      - checkout
+      - run: bash dash/scripts/dash-ci.sh Release
+      - run: bash dash/scripts/dash-ci.sh Minimal
+      - run:
+          command: bash dash/scripts/circleci/collect-artifacts.sh
+          when: always
+      - store_artifacts:
+          path: /tmp/build-logs
+      - store_test_results:
+          path: /tmp/build-tests
+
+  gnu_openmpi2:
+    docker:
+      - image: dashproject/ci:openmpi2
+    environment:
+      DASH_MAKE_PROCS: 2
+      DASH_MAX_UNITS: 3
```

Comment:
Can we set `DASH_MAX_UNITS` to 4? My gut tells me that there might be a) test that require 4 units, and b) some problems don't trigger with less than 4 units. Both are just feelings, though.

Reply:
Let's try that. In circle v1 this was not possible as the tests run incredibly slow.

Reply:
According to the artifacts, the topology stuff fails on CI. We should either disable this or run with just 3 units. Let's discuss this in the meeting.